dbus_service.py

```diff
@@ -176,12 +175,10 @@ def _read_config_dtu(self, actual_inverter):
         self.pvinverternumber = actual_inverter
         self.dtuvariant = str(config["DEFAULT"]["DTU"])
         if self.dtuvariant not in (constants.DTUVARIANT_OPENDTU, constants.DTUVARIANT_AHOY):
-            raise ValueError(
-                f"Error in config.ini: DTU must be one of \
+            raise ValueError(f"Error in config.ini: DTU must be one of \
                 {constants.DTUVARIANT_OPENDTU}, \
                 {constants.DTUVARIANT_AHOY}, \
```

Comment:
It's just syntax. I like to old style best but I don't mind if this gets changed. It will just be hard to remember this to have a common style

Reply:
This is an additional line which does not improve readability - in my eyes.